dump978-fa/start.sh

```diff
@@ -5,13 +5,15 @@ set -e
 
 if [[ ",$(echo -e "${DISABLED_SERVICES}" | tr -d '[:space:]')," = *",$BALENA_SERVICE_NAME,"* ]]; then
         echo "$BALENA_SERVICE_NAME is manually disabled."
+        curl --header "Content-Type:application/json" "$BALENA_SUPERVISOR_ADDRESS/v2/applications/$BALENA_APP_ID/stop-service?apikey=$BALENA_SUPERVISOR_API_KEY" -d '{"serviceName": "dump978-fa"}'
```

Comment:
Instead of dump978-fa here (and other places the same) you could just use $BALENA_SERVICE_NAME as above

Reply:
Good idea

Reply:
Adjusted :)